index.html

```diff
@@ -144,165 +260,228 @@ <h2 id="examples">Examples</h2>
 
 &lt;script src=&quot;analytics.js&quot; importance=&quot;low&quot;&gt;&lt;/script&gt;
       </pre>
-</div>
-
-<ul>
+    </div>
 
+    <ul>
 
-  <li>Blocking scripts are often of high priority (depends on their location in the page and other heuristics), yet sometimes
-    developers want to avoid them interfering with e.g., loading of viewport images.</li>
-
-  <li>When developers download a group of resources as a result of user interaction, those resources' download priorities don't
-    take into account the eventual usage and importance of those resources. Developers may wish to load these resources with
-    priorities and dependencies which better represent their usage and the user's needs.</li>
-
-  <li>Single-page applications can kick off multiple API requests to bootstrap the user experience. Developers may wish to load
-    critical API requests at a high priority and have better control over scheduling priority for the rest.</li>
-</ul>
-
-<h3 id="adoptionpath">Adoption path</h3>
 
-<p>Markup-based signals should be added in a way such that non-supporting browsers will simply ignore them and load all resources,
-  potentially not with the intended priority and dependency. Script-based signaling APIs should be created in a way that non-supporting
-  browsers simply ignore the signals.</p>
+      <li>Blocking scripts are often of high priority (depends on their location in the page and other heuristics), yet sometimes
+        developers want to avoid them interfering with e.g., loading of viewport images.</li>
```

Comment:
That's actually an example we stated where the browser should ignore the hints, as blocking scripts (at least if placed high in the document) are objectively more important